Doc/UK VATSIM frequencies.txt

```diff
@@ -61,6 +61,7 @@ Edinburgh Ground 121.755, 121.750 on VATSIM (8.33kHz-spaced frequency)
 Edinburgh Tower 118.705, 118.700 on VATSIM (8.33kHz-spaced frequency)
 Edinburgh Approach 121.205, 121.200 on VATSIM (8.33kHz-spaced frequency)
 Edinburgh Director 128.980, 128.970 on VATSIM (8.33kHz-spaced frequency)
+Exeter Tower 119.905, 119.900 on VATSIM (8.33kHz-spaced frequency)
```

Comment:
Hi Kai, also APP, DEL and RAD are 8.33 khz

Reply:
My mistake didnt read the full PR, updating

Reply:
No reference to DEL in the AMDT
![image](https://user-images.githubusercontent.com/46931474/90063617-9132cf80-dce1-11ea-8a42-c15794089441.png)